Assets/Scripts/AwakenessManager.cs

```diff
@@ -78,18 +78,12 @@ private void FixedUpdate()
     
     private float DistToClosestEnemy()
     {
-        GameObject[] enemies = GameObject.FindGameObjectsWithTag("Enemy");
-        float minDistance = Mathf.Infinity;
-        Vector3 position = transform.position;
-        foreach (GameObject enemy in enemies)
-        {
-            float curDistance = Vector3.Distance(enemy.transform.position, position);
-            if (curDistance < minDistance)
-            {
-                minDistance = curDistance;
-            }
-        }
-        return minDistance;
+        if (Enemy.Enemies.Count == 0) return Mathf.Infinity;
+
+        Vector3 playerPosition = transform.position;
+        Vector3 closestEnemyPosition = Enemy.ClosestEnemy(playerPosition).transform.position;
```

Comment:
The "ClosestEnemy" function seems to only be used here so... maybe we can just make this "ClosestEnemyDistance" or something, so we don't need the extra distance calculation below?

Reply:
Gotcha, updated